### DISCUSSION TITLE
正被消磨殆尽的耐心 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#172] [General]
# 正被消磨殆尽的耐心 - 愧怍的小站

在以前，我可以花费 4，5 个小时的时间专注于一件事情上，并且丝毫没有不耐烦的意思。而且这种状态能持续个好几天。例如拼个积木，拼个拼图，写个代码等等。专注个半天根本不成问题。

https://kuizuo.cn/patience-wearing-out/#comments

**Comment by NOeSIS-yu** [comment]
可以找找其他爱好，任何爱好成为工作后也会变味，而且我觉得没耐心是正常的，为什么要浪费自己的时间去帮别人呢？而且别人的反应是不可控的，只输出而没有碰撞久了就会感到无趣

**Reply by kuizuo** [reply]
确实，一年后回看这篇文章，感觉更多是压力所带来的不耐烦。之前是抱着娱乐游玩的心态，但如今确实抱着养家糊口的心态。实际印象的是心态，潜移默化地耐心就随之下降。
